jerry-debugger/jerry_client_main.py

```diff
@@ -1176,17 +1178,17 @@ def _process_scope_variables(self):
 
         while buff_pos != buff_size:
             # Process name
-            name_length = ord(self.scope_vars[buff_pos:buff_pos + 1])
+            name_length = int.from_bytes(self.scope_vars[buff_pos:buff_pos + 1].encode('latin1'))
```

Comment:
Why is this rewrite necessary? `self.scope_vars` is a string. `self.scope_vars[buff_pos:buff_pos + 1]` is a single-character string. Encoding it to latin1 gives a single-byte byte array containing the codepoint of the character. Then, the single byte is converted to an integer. How is this different from calling `ord()` on the character?

Notes:
- `self.scope_vars[buff_pos:buff_pos + 1]` can be simplified to `self.scope_vars[buff_pos]`.
- `int.from_bytes`, as used here, is not generic python3-compatible, as it needed a second argument up until python 3.10.

Reply:
It is not necessary.

Originally I removed all `ord` calls, which in hindsight was a mistake, then I reinvented the wheel without realizing that `ord` already did what was needed